portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -19,14 +19,90 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import java.util.*;
+
+import com.google.gson.Gson;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Query.SortDirection;
+
+
+
+
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
+  /**
+   * Define the object of a comment for message section.
+   * @param name The author of the comment.
+   * @param content The content of the comment.
+   * @param timestamp Automatically record the timestamp of the comment submit.
+   */
+  private class Comment{
+    private String name;
+    private String content;
+    private long timestamp;
```

Comment:
@param is only used to document method, not class.

```suggestion
  /** Define the object of a comment for message section. */
  private class Comment{
    /** The author of the comment */
    private String name;
    /** The content of the comment */
    private String content;
    /** The timestamp of the comment submission which records automatically */
    private long timestamp;
```